src/udev/udevadm-trigger.c

```diff
@@ -40,28 +40,29 @@ static int exec_list(struct udev_enumerate *udev_enumerate, const char *action,
         int r;
 
         udev_list_entry_foreach(entry, udev_enumerate_get_list_entry(udev_enumerate)) {
-                char filename[UTIL_PATH_SIZE];
-                const char *syspath;
-                int fd;
+                const char *filename, *syspath;
+                _cleanup_close_ int fd = -1;
 
                 syspath = udev_list_entry_get_name(entry);
                 if (verbose)
                         printf("%s\n", syspath);
                 if (dry_run)
                         continue;
 
-                strscpyl(filename, sizeof(filename), syspath, "/uevent", NULL);
+                filename = strjoina(syspath, "/uevent");
```

Comment:
stack allocations are not permissible in loops. And we are in  a loop here..

Reply:
I got ahead of myself. Fixed now.